aspnet/web-api/overview/web-api-routing-and-actions/routing-in-aspnet-web-api.md

```diff
@@ -77,38 +76,46 @@ Also, note that the POST request will fail, because the controller does not defi
 
 The previous section described the basic routing mechanism for ASP.NET Web API. This section describes some variations.
 
-### HTTP Methods
+### HTTP verbs
+
+Instead of using the naming convention for HTTP verbs, you can explicitly specify the HTTP verb for an action by decorating the action method with one of the following attributes:
 
-Instead of using the naming convention for HTTP methods, you can explicitly specify the HTTP method for an action by decorating the action method with the **HttpGet**, **HttpPut**, **HttpPost**, or **HttpDelete** attribute.
+- `[HttpGet]`
```

Comment:
Shouldn't the list be based on \* (asterisks) ... or was the convention changed at some point?

Reply:
The doc was already using dashes for other lists, so I stuck with that convention. Also, Markdownlint was flagging the asterisk-based lists as invalid in this particular doc.

Reply:
Cool ... just making sure that there wasn't an editorial change.